ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java

```diff
@@ -62,12 +61,26 @@ public void addPatientAppointmentTest() throws Exception {
 
     @After
     public void tearDown() throws Exception {
+    	removeWeekends();
         deletePatient(patient);
     }
+    
+    public void includeWeekends() {
+    	SystemAdministrationPage systemAdministrationPage = new SystemAdministrationPage(page);
+		systemAdministrationPage.activateWeekends("true");
+		homePage.goToPage("index.htm");	
+    }
+    
+    public void removeWeekends() {
+    	homePage.goToPage("index.htm");
```

Comment:
home page url should already be index.htm or home.page which should actually load the same thing